src/EVTUI/Core/DataManager.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace EVTUI;
+
+public class DataManager
+{
+
+    ////////////////////////////
+    // *** PUBLIC MEMBERS *** //
+    ////////////////////////////
+    public ProjectManager ProjectManager;
+    public EventManager EventManager;
+
+    public bool ReadOnly;
+    public bool ProjectLoaded;
+    public bool EventLoaded;
+
+    public List<string> CpkList { get; set; }
+    public string?      CpkPath
+    {
+        get
+        {
+            if (!this.ReadOnly && this.ProjectLoaded)
+                return this.ProjectManager.UserData.Projects[0].Immutable.Game.Path;
```

Comment:
Lol, I'll just explain this here for all the related comments: In short, no, it will usually be longer, but a loaded project will always be at the zero index.

In long, all of these methods using the zero index first do a check of `ProjectLoaded`. That gets set as true in `LoadProject`, which does so alongside a call of the method by the same name in the `ProjectManager` class. What that method does is move the project at the provided index, to the zero index. It does this to keep the user history in the right order -- so the most recently loaded project will always show up at the top of the table in the "Load Existing Project" config window. So... I use that as a shortcut to also avoid storing an extra index for which project is loaded... because said project is, by design, always at the zero index.

Hopefully that makes sense...? TL;DR: this is intentional and, IME, works as intended.

Reply:
As we discussed, I added some wrappers to make this clearer.